public/data/saugat.json

```
@@ -0,0 +1,26 @@
+{
+    "name": "Saugat Rajbhandari",
+    "type": "personal",
+    "bio": "Open Source Enthusiast!",
+    "avatar": "https://github.com/saugatrajbhandari.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/saugatrajbhandari",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/hey_saugat",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started College",
+        "date": "December 2019",
+        "icon": "dollar",
+        "color": "blue"
+      }
+    ]
+  }
```

Missing an endline. It's best to add it as it could cause issues with some IDEs and configurations.
```suggestion
}

```